integreat_cms/release_notes/current/unreleased/2246.yml

```
@@ -0,0 +1,2 @@
+en: Fix AttributeError in PDF export
+de: AttributeFehler beim PDF-Export beheben
```

```suggestion
en: Fix error in PDF export
de: Fehler beim PDF-Export beheben
```
The users never got to see the `AttributeError`, this is only shown in debug mode and in the server logs. As the release notes are targeted at the users of the system and not the developers, I think the error description should match what they experienced.